docs/glossary.md

```diff
@@ -259,7 +269,19 @@ _Tags: [term](/tags/term), [explanation](/tags/explanation)_
 
 #### [Multi-Cluster Replication](/self-hosted-guide/multi-cluster-replication)
 
-Multi-Cluster Replication is a feature which asynchronously replicates Workflow Executions from active Clusters to other passive Clusters, for backup and state reconstruction.
+Multi-Cluster Replication asynchronously replicate Workflow Executions and metadata from an active [Cluster](#temporal-cluster) to a standby Cluster.
```

Comment:
```suggestion
Multi-Cluster Replication asynchronously replicates Workflow Executions and metadata from an active [Cluster](#temporal-cluster) to a standby Cluster.
```

Might as well align on "Temporal Service" instead of "Temporal Cluster" as we are actively updating it through our content now.